src/GallerySwiper.jsx

```diff
@@ -1009,9 +936,24 @@ class GallerySwiper extends Component {
             }
         });
 
+        let events;
+        if (!disableArrowKeys) {
+            events = (
+                <AttachHandler
+                    target={BASE_CLASS}
```

Comment:
@sylvesteraswin any ideo how to create id names so there are no problems with multiple instances?